controllers/error-handling.controller.js

```diff
@@ -1,13 +1,12 @@
 exports.handlePSQL400s = (err, req, res, next) => {
-  console.log(err.code);
+  // console.log(err.code, err.status, err.msg);
   if (err.code === "22P02") {
-    res.status(err.status).send({ msg: err.msg });
+    res.status(400).send({ msg: "Invalid input" });
   } else {
     next(err);
   }
 };
 
 exports.handleCustomErrors = (err, req, res, next) => {
-  console.log(err);
   res.status(err.status).send({ msg: err.msg });
 };
```

Comment:
I'd take this file out of the controllers folder as it isn't technically a controller.
And make either an errors folder or have it in your root.